### ISSUE TITLE
Question: Is there a better was to do unifiedEvent handling in ember Octane?

### ISSUE THREAD

**Issue by edeis53** [trentmwillis/ember-singularity#28]
I noticed that this repo hasn't been updated recently (it's on the 2.x branch of ember), and just wanted to figure out if I should be using this add-on in a new octane app? 

Have recent developments in the 3.x branch made this addon redundant? Is there a way to easily do unifiedEvent handling in current ember without it?

**Comment by trentmwillis** [comment]
Hi @edeis53, unfortunately, I am no longer actively maintaining this package. I don't really do any Ember development these days so I am unsure what the strategy should be in Octane.

**Comment by Duder-onomy** [comment]
@edeis53 You are correct that ember-octane with element modifiers has awesome event support. I think this library is still useful as an event bus within your JS code. Just my opinion.
